app/javascript/types/resources.ts

```diff
@@ -1,10 +1,54 @@
 import type { Record } from 'immutable';
 
-type AccountValues = {
-  id: number;
+type CustomEmoji = Record<{
+  shortcode: string;
+  static_url: string;
+  url: string;
+}>
+
+type AccountField = Record<{
+  name: string;
+  value: string;
+  verified_at: string | null;
+}>
+
+type AccountApiResponseValues = {
+  acct: string;
   avatar: string;
   avatar_static: string;
-  [key: string]: any;
+  bot: boolean;
+  created_at: string;
+  discoverable: boolean;
+  display_name: string;
+  emojis: CustomEmoji[];
+  fields: AccountField[];
+  followers_count: number;
+  following_count: number;
+  group: boolean;
+  header: string;
+  header_static: string;
+  id: string;
+  last_status_at: string;
+  locked: boolean;
+  note: string;
+  statuses_count: number;
+  url: string;
+  username: string;
+};
+
+type NormalizedAccountField = Record<{
+  display_name_html: string;
+  note_emojified: string;
+  note_plain: string;
+}>;
```

Comment:
That doesn't sound right, shouldn't it extend `AccountField` with `name_emojified`, `value_emojified` and `value_plain`?

Reply:
fixed, thank you!